### PR TITLE
fix junit scope

### PR DESCRIPTION
**Pull request by Le-Zheng**
This pr is created to change `junit` scope to test.
local test passed.
ｊｕｎｉｔ

### CODE REVIEW

**Review by glorysdj — APPROVED** [review]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -152,13 +152,23 @@
...
+          <artifactId>junit</artifactId>
+	</exclusion>
       </exclusions>
     </dependency>
     <dependency>
       <groupId>org.codehaus.plexus</groupId>
       <artifactId>plexus-utils</artifactId>
       <version>3.0.16</version>
     </dependency>
+    <dependency>
```
Comment (qiuxin2012) [inline pom.xml:166]: submodule extends parent's dependency, maybe you can remove the junit in submodules or move this junit to from `dependency` to `dependencyManagement` (version management, same version in one pom)

Comment (Le-Zheng) [reply pom.xml]: updated.

Comment (Le-Zheng) [reply pom.xml]: remove version in submodule

**Pull request merged**